pkg/util/docker/containers.go

```diff
@@ -209,14 +209,14 @@ func parseContainerNetworkAddresses(ports []types.Port, netSettings *types.Summa
 				Port:     int(port.PublicPort), // Exposed port
 				Protocol: port.Type,
 			})
-		} else {
-			// Port is not exposed, cache it and retieve IPs from network settings
-			tempAddrList = append(tempAddrList, containers.NetworkAddress{
-				Port:     int(port.PrivatePort),
-				Protocol: port.Type,
-			})
 		}
+		// Cache container ports
+		tempAddrList = append(tempAddrList, containers.NetworkAddress{
+			Port:     int(port.PrivatePort),
```

Comment:
Is this intentionally missing an IP ?

Reply:
added in the next loop, from `netSettings.Networks`, the logic won't send tuples without IPs.